Source/driver/meth_params.template

```diff
@@ -17,6 +17,10 @@ module meth_params_module
 
   ! number of ghost cells for the hyperbolic solver
   integer, parameter     :: NHYP    = 4
+#ifdef MHD
+  integer, parameter     :: MAXADV  = 5
```

Comment:
I don't think `MAXADV` is used anywhere

Reply:
agree, we don't need it. At least grep suggests that :)